cmd/serverless/main.go

```diff
@@ -173,18 +149,6 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 	// KMS_ENCRYPTED or SECRET_ARN
```

Comment:
Can the code related to the API key be moved before calling `serverlessDaemon.Start(startTime)`? 
If yes, can the remaining code between `serverlessDaemon.Start(startTime)` and `serverlessDaemon.StartAgents(datadogConfigPath, serverlessID)` be moved into `serverlessDaemon.Start`? The idea behind this change is to have a single method to start serverless. What do you think?